tests/tests/ClassInitTests/PayloadTest.php

```diff
@@ -29,6 +29,25 @@
  */
 class PayloadTest extends TestCase {
 
+    public function testPayloadConstruct() {
+        $event_payload = new Payload();
+        $array = $event_payload->get();
+
+        // Asserts
+        $this->assertArrayHasKey("dtm", $array);
+        $this->assertTrue(isset($array["dtm"]));
+    }
+
+    public function testPayloadConstructWithTstamp() {
+        $event_payload = new Payload(1614072292000);
+        $array = $event_payload->get();
+
+        // Asserts
+        $this->assertArrayNotHasKey("dtm", $array);
```

Comment:
As mentioned above, events should always have a `dtm`. They may also have a `ttm`.